Automattic/jetpack
# PHPCS: handle PHPCS errors in API files, part 3

Fixes #21950

#### Changes proposed in this Pull Request:

* Get rid of more PHPCS linting errors in API files.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Not much change in actual functionality here, check for typos.
* One could check the Publicize feature in the block editor, since this PR touches that.
